src/Main/ProjectFile/ProjectManagerComponent.cpp

```diff
@@ -71,7 +71,7 @@ void ProjectManagerComponent::Save(QXmlStreamWriter* writer) {
 
   stream.writeStartElement(PROJECT_OPTION);
   stream.writeAttribute(PROJECT_NAME, PROJECT_CONFIG_TYPE);
-  stream.writeAttribute(PROJECT_VAL, tmpProCfg->projectType());
+  stream.writeAttribute(PROJECT_VAL, QString::number(tmpProCfg->projectType()));
```

Comment:
how about serializing project type by name, but not its number? 
we could make use of helper function converting project type to string (currently part of project_type_form).

Reply:
it is always better to use numbers and not strings